src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationClient.cs

```diff
@@ -154,32 +154,28 @@ Task IDurableOrchestrationClient.RaiseEventAsync(string taskHubName, string inst
         }
 
         /// <inheritdoc />
-        Task IDurableOrchestrationClient.SignalActor(string taskHubName, ActorId actorId, string operationName, object operationContent, string connectionName)
+        Task IDurableOrchestrationClient.SignalActor(ActorId actorId, string operationName, object operationContent, string taskHubName, string connectionName)
         {
-            if (string.IsNullOrEmpty(connectionName))
+            if (string.IsNullOrEmpty(taskHubName))
```

Comment:
Would it be more correct to check for both a null/empty `taskHubName` **and** `connectionName`?